packages/ubuntu_provision/lib/src/eula/eula_page.dart

```diff
@@ -1,15 +1,35 @@
 import 'dart:io';
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:intl/intl.dart';
+import 'package:intl/intl_standalone.dart';
+import 'package:path/path.dart';
 import 'package:pdfrx/pdfrx.dart';
 import 'package:ubuntu_logger/ubuntu_logger.dart';
 import 'package:ubuntu_provision/interfaces.dart';
-import 'package:ubuntu_provision/providers.dart';
 import 'package:ubuntu_provision/src/eula/eula_l10n.dart';
 import 'package:ubuntu_utils/ubuntu_utils.dart';
 import 'package:ubuntu_wizard/ubuntu_wizard.dart';
 import 'package:yaru/yaru.dart';
 
+final eulaPageProvider = FutureProvider<File>((ref) async {
+  final selectedLocale = Locale(Intl.defaultLocale ?? await findSystemLocale());
+  final fileList = Directory('/usr/share/desktop-provision/eula').listSync();
+  final exp = RegExp(r'EULA_([a-zA-Z0-9\-]+)\.');
+  final supportedLocales = fileList
+      .map((entry) => basename(entry.path))
+      .where(exp.hasMatch)
+      .map((entry) => Locale(exp.firstMatch(entry)!.group(1)!));
+
+  final locale = basicLocaleListResolution([selectedLocale], supportedLocales);
+  final eulaFile = File(
+      '/usr/share/desktop-provision/eula/EULA_${locale.toLanguageTag()}.pdf');
```

Comment:
This results in a different format than for the slides, I would say it is better to be consistent here and use the same format. We should also specify the format better in the customization guide.